js_task1/secondTask.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+    <section class="second-task">
+        <a href="./index.html">
+            Back
+        </a>
+        <h1>
+            Second Task
+        </h1>
+        <button class="generate-btn">
+            Generate
+        </button>
+        <button class="check-btn">
+            Check!
+        </button>
+    </div>
```

Comment:
I think it should be 'section' for as close tag. Please check the same in other failes.

Reply:
Stupid "auto rename tag" extention)